examples/todos/package-lock.json

```diff
@@ -0,0 +1,1315 @@
+{
```

Comment:
Need to remove this file. Not sure how it showed up as we have a `.npmrc` file with `package-lock=false
`

Reply:
@andrewvc Can you remove this file?